tests/__sf_section_vi_mode.test.fish

```diff
@@ -0,0 +1,143 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+
+function setup
+	spacefish_test_setup
+	fish_vi_key_bindings
```

Comment:
It appears CI is failing in the setup step. After some troubleshooting locally, it looks like fishtape thinks setup didn't end with a status 0 for whatever reason. 🤔

I ended up getting it to work locally with the following:

```suggestion
	fish_vi_key_bindings; or true
```